youtube_dl/extractor/kick.py

```diff
@@ -0,0 +1,46 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
```

Comment:
If the other suggestions are used, you'll need to `from ..utils import (`... a load of things.